app/ceom/photos/views.py

```diff
@@ -18,11 +18,11 @@
 from django.core.files.base import ContentFile
 from django.core.files import File
 from django.shortcuts import redirect
-from django.contrib.auth.models import User
 
 
 from django.views.decorators.csrf import csrf_exempt
 from django.template.defaultfilters import wordwrap
+from ceom.photos.models import PhotoUser
```

Comment:
Move this import down a couple lines such that the `ceom.whatever` imports are all grouped together